packages/uniorg-rehype/src/org-to-hast.ts

```diff
@@ -258,7 +258,7 @@ export function orgToHast(
         return h(
           org,
           `h${org.level}`,
-          {},
+          {todo ? "todo-item" : ""},
```

Comment:
The value needs to be assigned to the `className` field (otherwise, it's invalid syntax):
```suggestion
          { className: todo ? 'todo-headline' : '' },
```